docker-compose.yml

```diff
@@ -0,0 +1,20 @@
+version: '3'
+
+services:
+  springboot-docker-compose-app-container:
+    image: springboot-docker-compose-app:1
```

Comment:
we need to use parametrization for server deployment, meaning to make use of kustomize like in europeana apis
see translation API for example:
https://github.com/europeana/translation-api/blob/develop/k8s/base/deployment.yaml
https://github.com/europeana/translation-api/blob/develop/k8s/base/kustomization.yaml
https://github.com/europeana/translation-api/blob/develop/k8s/overlays/cloud/deployment_patch.yaml.template
https://github.com/europeana/translation-api/blob/develop/k8s/overlays/cloud/kustomization.yaml